core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/StackV4Controller.java

```diff
@@ -327,6 +331,12 @@ public FlowIdentifier upgradeClusterByNameInternal(Long workspaceId, String name
         return stackUpgradeOperations.upgradeCluster(NameOrCrn.ofName(name), restRequestThreadLocalService.getRequestedWorkspaceId(), imageId);
     }
 
+    @Override
+    @InternalOnly
+    public FlowIdentifier upgradeCcmByCrnInternal(Long workspaceId, String name, String initiatorUserCrn) {
```

Comment:
missing annotation from parameter (https://github.com/hortonworks/cloudbreak/tree/master/authorization-common#how-to-make-an-api-internally-callable)